pkg/controller/querier/querier.go

```diff
@@ -28,24 +29,32 @@ const (
 
 var _ ControllerQuerier = new(controllerQuerier)
 
+//TODO: expand this interface similarly to AgentQuerier
 type ControllerQuerier interface {
 	GetControllerInfo(controllInfo *v1beta1.AntreaControllerInfo, partial bool)
 }
 
+//TODO: implement interface methods
 type controllerQuerier struct {
 	networkPolicyInfoQuerier querier.ControllerNetworkPolicyInfoQuerier
+	endpointQuerier          networkpolicy.EndpointQuerier
 	apiPort                  int
 }
 
-func NewControllerQuerier(networkPolicyInfoQuerier querier.ControllerNetworkPolicyInfoQuerier, apiPort int) *controllerQuerier {
-	return &controllerQuerier{networkPolicyInfoQuerier: networkPolicyInfoQuerier, apiPort: apiPort}
+func NewControllerQuerier(networkPolicyInfoQuerier querier.ControllerNetworkPolicyInfoQuerier,
+	endPointQuerier networkpolicy.EndpointQuerier, apiPort int) *controllerQuerier {
+	return &controllerQuerier{networkPolicyInfoQuerier: networkPolicyInfoQuerier,
+		endpointQuerier: endPointQuerier, apiPort: apiPort}
 }
 
-// GetNodeSubnet gets current network policy info querier.
 func (cq controllerQuerier) getNetworkPolicyInfoQuerier() querier.ControllerNetworkPolicyInfoQuerier {
 	return cq.networkPolicyInfoQuerier
 }
 
+func (cq controllerQuerier) getEndpointQuerierReplier() networkpolicy.EndpointQuerier {
+	return cq.endpointQuerier
+}
+
```

Comment:
with the current state of the code, we actually do not use this anywhere. So we should either remove your changes to this file, or have a small refactoring in cmd/antrea-controller/controller.go to leverage this.

The easiest solution is definitely to just remove this code, so I suggest we go this route. Keep this code somewhere in another branch (rebase-2 is fine) in case we need it later. In particular, I think the refactoring would involve moving `createAPIServerConfig` to `pkg/apiserver`, and making the instantiation of the Agent apiserver more similar to the instantiation of the Controller apiserver. Probably a bit of a tricky project at this stage.

Reply:
I was also confused by the TODOs, but it's irrelevant if we just remove the code

Reply:
I do need to include the changes in NewControllerQuerier correct?

Reply:
Actually never mind, it is never used